packages/discord.js/src/structures/GuildScheduledEvent.js

```diff
@@ -189,6 +189,56 @@ class GuildScheduledEvent extends Base {
     } else {
       this.image ??= null;
     }
+
+    /**
+     * Represents the recurrence rule for a {@link GuildScheduledEvent}.
+     * @typedef {Object} GuildScheduledEventRecurrenceRule
+     * @property {number} startTimestamp The timestamp the recurrence rule interval starts at
+     * @property {Date} startAt The time the recurrence rule interval starts at
+     * @property {?number} endTimestamp The timestamp the recurrence rule interval ends at
+     * @property {?Date} endAt The time the recurrence rule interval ends at
+     * @property {GuildScheduledEventRecurrenceRuleFrequency} frequency How often the event occurs
+     * @property {number} interval The spacing between the events
+     * @property {?(GuildScheduledEventRecurrenceRuleWeekday[])} byWeekday The days within a week to recur on
+     * @property {?(GuildScheduledEventRecurrenceRuleNWeekday[])} byNWeekday The days within a week to recur on
+     * @property {?(GuildScheduledEventRecurrenceRuleMonth[])} byMonth The months to recur on
+     * @property {?(number[])} byMonthDay The days within a month to recur on
+     * @property {?(number[])} byYearDay The days within a year to recur on
```

Comment:
I don't think the `()` are needed

```suggestion
     * @property {?GuildScheduledEventRecurrenceRuleWeekday[]} byWeekday The days within a week to recur on
     * @property {?GuildScheduledEventRecurrenceRuleNWeekday[]} byNWeekday The days within a week to recur on
     * @property {?GuildScheduledEventRecurrenceRuleMonth[]} byMonth The months to recur on
     * @property {?number[]} byMonthDay The days within a month to recur on
     * @property {?number[]} byYearDay The days within a year to recur on
```